cloud/amazon/ec2_elb_lb.py

```diff
@@ -360,6 +360,29 @@
 except ImportError:
     HAS_BOTO = False
 
+import time
+import random
+
+def _throttleable_operation(max_retries):
+    def _operation_wrapper(op):
+        def _do_op(*args, **kwargs):
+            retry = 0
+            while True:
+                try:
+                    return op(*args, **kwargs)
+                except boto.exception.BotoServerError, e:
```

Comment:
Actually... no. That syntax isn't Python 2.4 compatible, and while boto requires 2.6 and thus would support the "as" syntax, the Ansible team would prefer to use consistent style across all their projects. Either use the "," syntax, or there's some other methods in module_utils.basic that try to do this in an independent way, but the "," syntax is generally pretty standard.